src/main/webapp/i18n/en/fileUploadExercise.json

```diff
@@ -7,6 +7,8 @@
                 "editLabel": "Edit File Upload Exercise",
                 "importLabel": "Import File Upload Exercise"
             },
+            "create": "Create File Upload",
+            "import": "Import File Upload",
```

Comment:
I think it should contain "exercise" and we should not duplicate translations keys